tests/tests/da_compression.rs

```diff
@@ -66,9 +108,24 @@ async fn can_fetch_da_compressed_block_from_graphql() {
         }
     };
 
-    let block = client.da_compressed_block(block_height).await.unwrap();
-    let block = block.expect("Unable to get compressed block");
-    let _: VersionedCompressedBlock = postcard::from_bytes(&block).unwrap();
+    let block = client
+        .da_compressed_block(block_height)
+        .await
+        .unwrap()
+        .expect("Unable to get compressed block");
+    let block: VersionedCompressedBlock = postcard::from_bytes(&block).unwrap();
+
+    let db = &srv.shared.database;
+    let mut tx_inner = db.off_chain().clone().into_transaction();
+    let db_tx = DecompressDbTx {
+        db_tx: DbTx {
+            db_tx: &mut tx_inner,
+        },
+        onchain_db: db.on_chain().view_at(&0u32.into()).unwrap(),
```

Comment:
It would be nice if in the test you highlighted somehow that we use the previous state of the database because it is important detail=)

Reply:
Added a comment to that effect in aca530adcb